code/modules/projectiles/guns/projectile/constructable/flamethrower.dm

```diff
@@ -205,6 +205,8 @@
 		onclose(user, "flamethrower", src)
 
 /obj/item/weapon/gun/projectile/flamethrower/Topic(href,href_list[])
+	if (..())
+		return
```

Comment:
I *believe* this should be `return 1`, but you may want to ask someone better-versed in `Topic()` to make sure